_topic_maps/_topic_map.yml

```diff
@@ -1958,8 +1958,8 @@ Topics:
     File: run-gitops-control-plane-workload-on-infra-nodes
   - Name: Sizing requirements for GitOps Operator
     File: about-sizing-requirements-gitops
-#  - Name: Collecting debugging data for a support case
-#    File: collecting-debugging-data-for-support
+  - Name: Collecting debugging data for a support case
+    File: collecting-debugging-data-for-support
```

Comment:
Uncommented the assembly for Shipra's PR: https://github.com/openshift/openshift-docs/pull/60608